docs/current/cookbook/snippets/filesystem-operations/add-git-dir-exclude-include/index.mjs

```diff
@@ -0,0 +1,26 @@
+import { connect } from "@dagger.io/dagger"
+
+// create Dagger client
+connect(
+  async (client) => {
+    // get repository at specified branch
+    const project = client
+      .git("https://github.com/dagger/dagger")
+      .branch("main")
+      .tree()
+
+    // return container with repository
+    // at /src path
+    // including all files except files beginning with .git
+    const contents = await client
+      .container()
+      .from("alpine:latest")
+      .withDirectory("/src", project, { include: ["*"], exclude: [".git*"] })
```

Comment:
It might be better to use a different example than `.git*` because someone could interpret the goal as "excluding the `.git` directory," which is actually [determined by `keepGitDir`](https://play.dagger.cloud/playground/JG8IFHpkBK7) which already defaults to `false`.

Just don't want folks leaving with the impression that they have to exclude the `.git` directory for more efficiency.

Not a huge deal though.

---

This also raises the question of whether `include: ["*"]` excludes hidden files, which might be worth explicitly calling out somewhere. I'm not actually sure whether it does! It would in the shell, but it depends on the globbing implementation. If it _does_, that means it's redundant with `exclude: [".git*"]`.

Reply:
Modified example.

Reply:
include: ["*"] doesn't seem to exclude hidden files